example-app/app/core/components/sidenav.component.ts

```diff
@@ -1,9 +1,9 @@
-import { Component, Input } from '@angular/core';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
   selector: 'bc-sidenav',
   template: `
-    <mat-sidenav [opened]="open">
+    <mat-sidenav #sidenav [opened]="open" (keydown.escape)="sidenav.close()" (closedStart)="closeMenu.emit()" disabelClose>
```

Comment:
@brandonroberts,
> disabelClose -> disableClose

fixed this typo

> Why not just call `closeSidenav.emit()` on both events?

I think this does not work, because we have to explicitly call the "closing" event of the menu when escape was pressed on the focused element. Otherwise it would never close since `disableClose` is set.